pkg/tsdbctl/create.go

```diff
@@ -62,6 +63,8 @@ func newCreateCommandeer(rootCommandeer *RootCommandeer) *createCommandeer {
 
 	cmd.Flags().StringVarP(&commandeer.defaultRollups, "aggregates", "a", "",
 		"Default aggregates to calculate in real time during\nthe samples ingestion, as a comma-separated list of\nsupported aggregation functions - count | avg | sum |\nmin | max | stddev | stdvar | last | rate.\nExample: \"sum,avg,max\".")
+	cmd.Flags().StringVarP(&commandeer.crossLabelSets, "cross-label", "l", "",
+		"Label sets for which cross-label pre-aggregations should be created. Must be used in conjunction with -a.\nExample: \"label1,label2.label3.\".")
```

Comment:
I find the dot non-intuitive. WDYT of "label1,label2||label3" ?

Reply:
Agreed to use semicolon instead of dot.